KeyVaultCA/KeyVaultCA.csproj

```diff
@@ -8,10 +8,24 @@
 
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.8.0" />
```

Comment:
Do we still need this?

Reply:
This reference is needed for `AddCommandLine(args)`, to be able to overwrite the settings with command line arguments.

Reply:
got it, thanks!